src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -74,18 +74,19 @@ export const GuidePanel = ({ api, application }: GuidePanelProps) => {
   const handleStepButtonClick = async (step: GuideStepStatus, stepConfig: StepConfig) => {
     if (guideState) {
       const { id, status } = step;
+
       if (status === 'ready_to_complete') {
         return await api.completeGuideStep(guideState?.guideId, id);
       }
 
-      if (status === 'active') {
```

Comment:
AFAICT we should call this when the step is `active` or `in_progress`, but it's possible I missed something 😅  I think this logic was originally added in https://github.com/elastic/kibana/pull/142884.

Reply:
yes, that was a regression from my PR, thanks for fixing this! 👍